introspection-engine/connectors/sql-introspection-connector/src/re_introspection.rs

```diff
@@ -50,6 +51,22 @@ pub fn enrich(
     });
 }
 
+fn keep_index_ordering(old_data_model: &Datamodel, new_data_model: &mut Datamodel) {
+    for old_model in old_data_model.models() {
+        let new_model = match new_data_model.models_mut().find(|m| m.name == *old_model.name()) {
+            Some(m) => m,
+            None => continue,
+        };
+
+        new_model.indices.sort_by(|idx_a, idx_b| {
+            let idx_a_idx = old_model.indices.iter().position(|idx| idx.db_name == idx_a.db_name);
+            let idx_b_idx = old_model.indices.iter().position(|idx| idx.db_name == idx_b.db_name);
+
+            re_order_putting_new_ones_last(idx_a_idx, idx_b_idx)
```

Comment:
are other calls to this function in other places still necessary then?

Reply:
The model and enum orders?

Reply:
the function looks like it's only about indexes